index.js

```diff
@@ -1,41 +1,43 @@
-const { combineReducers, createStore } = require('redux')
+const { createStore } = require('redux')
 const { Set } = require('immutable')
-const assert = require('assert');
-
-
+const assert = require('assert')
 
 const bookReducer = (state, action) => {
   if (state === undefined) {
-    state = { books: new Set(), customers: new Set()}
+    state = {books: new Set(), customers: new Set()}
   }
   if (action.type === 'ADD_BOOK') {
-    return { books: state.books.add(action.title), customers: state.customers}
+    return {books: state.books.add(action.title), customers: state.customers}
```

Comment:
This approach and verbose and there's a risk of forgetting to add new entries to each existing action handling.

Why not using a `immutable/Map` for the state instead?